holopy/core/tests/test_utils_math.py

```diff
@@ -29,17 +30,136 @@
 
 from holopy.core.utils import (
     ensure_array, ensure_listlike, mkdir_p, choose_pool)
-from holopy.core.math import rotate_points, rotation_matrix
+from holopy.core.math import (
+    rotate_points, rotation_matrix, transform_cartesian_to_spherical,
+    transform_spherical_to_cartesian, transform_cartesian_to_cylindrical,
+    transform_cylindrical_to_cartesian, find_transformation_function,
+    keep_in_same_coordinates)
 from holopy.core.tests.common import assert_obj_close, get_example_data
 
 
+TOLS = {'atol': 1e-14, 'rtol': 1e-14}
+
+class TestCoordinateTransformations(unittest.TestCase):
+    @attr("fast")
+    def test_transform_cartesian_to_spherical_returns_correct_shape(self):
+        np.random.seed(12)
+        xyz = np.random.randn(3, 10)
```

Comment:
holograms often have x & y in a grid but a single value for z, so (x, y, z) will have lenghts (N, N, 1). This is why there was previously a call to the function `repeat_sing_dims`. You should add a test to confirm that this case still works unless it has been taken care of elsewhere.

Reply:
it looks like this might be covered below in `test_detector_points` but I'm not positive.

Reply:
Should be fixed by 1c34846